services/rds/rds.go

```diff
@@ -60,6 +60,9 @@ const (
 
 	// maximum time for connecting to the database and running all queries
 	sqlCheckTimeout = 5 * time.Second
+
+	// maximum number for page to DescribeDBInstances
+	maxPageNum = 5
```

Comment:
Minimal value is 20: https://docs.aws.amazon.com/sdk-for-go/api/service/rds/#DescribeDBInstancesInput

Reply:
page -> pagination